### DISCUSSION TITLE
Preload / Prefetch Screens dynamically before navigate to them

### DISCUSSION THREAD

**Discussion by topkat** [react-navigation/react-navigation#11857] [General]
I noticed that the first time I navigate to a screen, it takes way too much time compared to the second time, where it is very quick.

Overall I think performances may gain a lot to be treated in the documentation.

Have you got any clue on how achieve that ?

Note: I have looked at #9903 but I am not sure how to implement it, and it don't seem like a generic solution

**Comment by satya164** [comment]
Preloading screens feature is planned for React Navigation 7.
